assets/wire/components/fields/DurationLabel.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {gettext} from 'utils';
+
+export function DurationLabel ({item, filterGroupLabels}: any) {
```

Comment:
@dzonidoo Please add types as per the PR checklist. This includes adding `duration` to the `IArticle['extra']` (https://github.com/superdesk/newsroom-core/blob/develop/assets/interfaces/content.ts#L26), and avoiding use of `any` and `PropType` and instead use the appropriate `IProps` interface to be used for this component